source/game.player.bmx

```diff
@@ -339,14 +339,19 @@ Type TPlayer extends TPlayerBase {_exposeToLua="selected"}
 	End Method
 
 
-	Method GetMaxAudience:Int() {_exposeToLua}
```

Comment:
Das veränderte KI-Verhalten bezüglich der Werbeverträge lässt sich auf diese Umbenennung zurückführen. In PlayerBase existieren noch die alten Methoden und liefern immer 0 zurück.
Umbenennung in PlayerBase und bei der Verwendung in Lua nötig.

Reply:
Vorgeschlagene Änderungen sind in meinem Branch.